src/object/cli_ec.c

```diff
@@ -1879,7 +1877,7 @@ obj_ec_fetch_set_sgl(struct dc_object *obj, struct obj_reasb_req *reasb_req,
 	stripe_rec_nr = obj_ec_stripe_rec_nr(oca);
 	cell_rec_nr = obj_ec_cell_rec_nr(oca);
 	uiods = reasb_req->orr_uiods;
-	usgls = reasb_req->orr_usgls;
+	usgls         = reasb_req->orr_processed_sgls;
```

Comment:
I feel confuse here, why use orr_processed_sgls
it need to set info in user's input sgl right?